### PR TITLE
adding Sieve of Eratosthenes program

### PR DESCRIPTION
**Pull request by chennakeshava1998**
#805 adding sieve of eratosthenes program

### CODE REVIEW

File: chennakeshava/sieve.cpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#include<bits/stdc++.h>
```
Comment (MJ10) [inline chennakeshava/sieve.cpp]: Please move this file to the `sieve` directory and rename it to your github username

Comment (chennakeshava1998) [reply chennakeshava/sieve.cpp]: ok, done.
#805 adding sieve of eratosthenes program

Comment (MJ10) [reply chennakeshava/sieve.cpp]: Please delete this folder as well.

Comment (chennakeshava1998) [reply chennakeshava/sieve.cpp]: I have deleted the original folder in which it was present ie chennakeshava

Comment (MJ10) [reply chennakeshava/sieve.cpp]: Did you push the changes after that? Because the folder is still visible here.

**Comment by MJ10** [comment]
@chennakeshava1998 Any updates?

**Comment by chennakeshava1998** [comment]
I have pushed the latest commit

**Pull request merged**